detector/detector.go

```diff
@@ -270,7 +270,7 @@ func buildLicenceRegex() *regexp.Regexp {
 	// inspired by https://github.com/src-d/go-license-detector/blob/7961dd6009019bc12778175ef7f074ede24bd128/licensedb/internal/investigation.go#L29
 	licenceFileNames := []string{
 		`li[cs]en[cs]es?`,
-		`license.(mit|apache)`,
+		`license.[a-z]{2,}`,
```

Comment:
IIUC https://github.com/spdx/tools-golang holds 2 files that would match `lisense.[a-z]{2,} `:

* [LICENSE.code](https://github.com/spdx/tools-golang/blob/main/LICENSE.code): the Apache License
* [LICENSE.docs](https://github.com/spdx/tools-golang/blob/main/LICENSE.docs): CC-BY-4.0

How do we know which one to select in that case? 
Would it not make sense [to create an override](https://github.com/elastic/go-licence-detector#adding-overrides) instead?

Reply:
Thank you @barkbay .
I opened the PR with the changes you suggested https://github.com/elastic/beats/pull/34902.
Once it's merged I will close this one.